tests/phpunit/api/v3/PaymentTest.php

```diff
@@ -181,7 +181,7 @@ public function testPaymentEmailReceipt() {
     $this->callAPISuccess('Payment', 'sendconfirmation', ['id' => $payment['id']]);
     $mut->assertSubjects(['Payment Receipt - Annual CiviCRM meet']);
     $mut->checkMailLog([
-      'Dear Anthony,',
+      'Dear Anthony',
```

Comment:
hmm - the comma shouldn't be disappearing - still I think this is the same template I said we shouldn't change

Reply:
I though it would serve consistency as most tests do not have the comma. Would you suppose the other way around? Add the comma to the other tests?